src/lib/entropy/darwin_secrandom/info.txt

```diff
@@ -0,0 +1,17 @@
+define ENTROPY_SRC_DARWIN_SECRANDOM 20150925
+
+<source>
+darwin_secrandom.cpp
+</source>
+
+<header:internal>
+darwin_secrandom.h
+</header:internal>
+
+<os>
+darwin
+</os>
+
+<frameworks>
+darwin -> Security
```

Comment:
What are the implications of this for users of shared or static libraries? For a shared lib I'd imagine it gets pulled in like any other shlib dependency, but will a user of a static library need to add `-framework Security` for their build on OS X with this enabled?